pyproj/geod.py

```diff
@@ -273,13 +280,23 @@ def npts(
         lat2: float,
         npts: int,
         radians: bool = False,
-    ) -> List:
+        initial_idx: int = 1,
+        terminus_idx: int = 1,
+        out_lons: Any = None,
+        out_lats: Any = None,
+        return_points: bool = True,
+    ) -> Union[List[Tuple[float, float]], Tuple[Sequence[float], Sequence[float]]]:
         """
+        .. versionadded:: 3.1.0 initial_idx, terminus_idx
+                                out_lons, out_lats, return_points,
+
         Given a single initial point and terminus point, returns
         a list of longitude/latitude pairs describing npts equally
         spaced intermediate points along the geodesic between the
         initial and terminus points.
 
+        Similar to inv_intermediate(), but with less options.
```

Comment:
With the addition of `inv_intermediate`, do we need the `return_points` kwarg?

Reply:
I don't mind as I will indeed use `inv_intermediate` instead.

I also thought about removing the redundant `_geod._npts` (and implemente `geod.npts` on top of `_inv_or_fwd_intermediate`, as it starts with `_` I understand it's not part of the official API - should I remove it ?

Reply:
Either way is fine. I will leave that decision up to you.